docs/targets/zendesk.md

```diff
@@ -1,10 +1,13 @@
 # Event Target for Zendesk
 
-This event target receives [CloudEvents][ce] and uses the Zendesk API to either create a new ticket or tag an existing one.
+This event Target receives [CloudEvents][ce] and uses the Zendesk API to either create a new ticket or tag an existing one.
 
-## Prerequisites
+## Prerequisite(s)
 
-A Zendesk API token is required to utilize this Target. The steps to obtain a token are outlined in the [Zendesk API Docs](https://support.zendesk.com/hc/en-us/articles/226022787-Generating-a-new-API-token-)
+- Zendesk API token (steps to obtain a token are outlined in the [Zendesk API Docs](https://support.zendesk.com/hc/en-us/articles/226022787-Generating-a-new-API-token-))
```

Comment:
Wondering if this should be just unordered list and more details below in its own section

Reply:
Agreed, will separate it.